realsense2_camera/src/t265_realsense_node.cpp

```diff
@@ -144,3 +165,69 @@ void T265RealsenseNode::warningDiagnostic(diagnostic_updater::DiagnosticStatusWr
 {
   status.summary(diagnostic_msgs::DiagnosticStatus::WARN, _T265_fault);
 }
+
+bool T265RealsenseNode::importLocalizationMap(const std::string &localization_file)
+{
+    try
+    {
+      _pose_sensor.import_localization_map(this->bytesFromRawFile(localization_file));
+      ROS_INFO_STREAM("T265 Localization map loaded from " << localization_file);
+    }
+    catch (std::runtime_error& e)
+    {
+      ROS_WARN_STREAM("Error loading T265 map from " << localization_file << ": " << e.what());
+      return false;
+    }
+    return true;

```

Comment:
this was inspired from https://github.com/IntelRealSense/librealsense/blob/master/examples/ar-advanced/rs-ar-advanced.cpp